WordPress/WordPressTest/PostTests.swift

```diff
@@ -376,6 +376,20 @@ class PostTests: XCTestCase {
         XCTAssertFalse(revision.hasLocalChanges())
     }
 
+    func testThatVersionConflictWorks() {
```

Comment:
I'm not a big fan of multiple tests inside a single function.

But I can see that there's a lot of others here like that. 😬

So it's up to you to make those changes or not.